Dj182drf313

thenewguy opened the pull request:
This pull request modifies the TagListSerializerField such that it works with the browsable api.

It also fixes a problem where tags could only be added and not removed.

thenewguy committed:
support the browsable api.  previously the tag field output looked like
[u'tag1', u'tag2'] and was not compatible with create/update

thenewguy committed:
use "set" instead of "add" so that tags can also be removed via the api

thenewguy committed:
do not use variable name "object"

thenewguy committed:
improve validation

thenewguy committed:
fix py3 basestring reference

thenewguy committed:
six

thenewguy committed:
pretty print json tag list in browsable api by default

glemmaPaul commented on taggit_serializer/serializers.py:
```diff
@@ -1,15 +1,77 @@
-
-class TagListSerializerField(serializers.ListField):
+from django.utils.translation import ugettext_lazy as _
+import json
```
Could you move the `json` on the top, so the imports would look like:

``` python
# -*- coding: utf-8 -*-
import json

# Third party
import six
from django.utils.translation import ugettext_lazy as _
```

thenewguy committed:
Reorder imports per
https://github.com/glemmaPaul/django-taggit-serializer/pull/14#discussion-diff-32948833

thenewguy commented:
I updated import ordering

glemmaPaul commented:
Closes #13
